python/socketioclient/socketio_client.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import base64
+import hashlib
+import hmac
+import ssl
+import time
+import uuid
+import logzero
+import socketio
+
+logger = logzero.logger
+
+ctx = ssl.create_default_context()
+ctx.set_ciphers('DEFAULT')
+
+URL = 'http://localhost:8080'
```

Comment:
change the url to https://stream.falconx.io?

Reply:
Done!